branca/utilities.py

```diff
@@ -44,11 +44,12 @@ def legend_scaler(legend_values, max_labels=10.0):
     if len(legend_values) < max_labels:
         legend_ticks = legend_values
     else:
-        spacer = int(math.ceil(len(legend_values)/max_labels))
+        spacer = int(math.ceil(len(legend_values) / (max_labels - 1)))
         legend_ticks = []
         for i in legend_values[::spacer]:
             legend_ticks += [i]
             legend_ticks += ['']*(spacer-1)
```

Comment:
Not related to your change directly, but I don't understand why we add empty strings in between. There doesn't seem to be a need for it. If the idea is to make the length of `legend_ticks` correspond to `legend_values`, it's not even correct since we add one more tick than value.

Reply:
Don't know but commenting it out doesn't change anything apparent.